osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/DlmsObjectConfigService.java

```diff
@@ -53,7 +53,7 @@ public DlmsObjectConfigService(final DlmsHelper dlmsHelper, final List<DlmsObjec
     public Optional<AttributeAddressForProfile> findAttributeAddressForProfile(final DlmsDevice device,
             final DlmsObjectType type, final Integer channel, final DateTime from, final DateTime to,
             final Medium filterMedium) {
-        return this.findDlmsObject(Protocol.withNameAndVersion(device.getProtocol(), device.getProtocolVersion()), type,
+        return this.findDlmsObject(Protocol.forDevice(device), type,
```

Comment:
Replaced cludgy convenience method `withNameAndVersion` with the more compact `forDevice`.